test/videojs-flashls-source-handler.test.js

```diff
@@ -35,3 +36,75 @@ QUnit.module('videojs-flashls-source-handler', {
   }
 });
 
+QUnit.test('Flashls Audiotracks tests', function(assert) {
+  const swfAudioTracks = [
+    {
+      id: 0,
+      enabled: true,
+      kind: '',
+      label: 'English',
+      language: ''
+    },
+    {
+      id: 1,
+      enabled: false,
+      kind: '',
+      label: 'Spanish',
+      language: ''
+    },
+    {
+      id: 2,
+      enabled: false,
+      kind: '',
+      label: 'French',
+      language: ''
+    }
+  ];
+
+  let vjsAudioTracks = [];
+  let swfAudioTrack = 0;
+
+  const tech = makeMochTech({
+    audioTracks: () => swfAudioTracks,
+    audioTrack: () => swfAudioTrack
+  }, {
+    audioTrack: (val) => swfAudioTrack = val
+  });
+
+  tech.audioTracks = () => {
+    return vjsAudioTracks;
+  };
+
+  tech.audioTracks().addTrack = (t) => {
+    vjsAudioTracks.push(t);
+  };
+
+  assert.equal(vjsAudioTracks.length, 0, 'Initial size of vjsAudioTracks');
+
+  setupAudioTracks(tech);
+
+  assert.equal(vjsAudioTracks.length, 3, 'Length after setupAudioTracks()');
+  assert.equal(vjsAudioTracks[0].id, '0', 'corrrect id');
+  assert.equal(vjsAudioTracks[0].enabled, true, 'correct audio track enabled');
+  assert.equal(vjsAudioTracks[1].enabled, false, 'Other Audio track are disabled');
+  assert.equal(vjsAudioTracks[2].enabled, false, 'Other Audio track are disabled');
+
+  vjsAudioTracks[0].enabled = false;
+
+  updateAudioTrack(tech);
```

Comment:
since the `swfAudioTrack` that is `0` is the default enabled track, I think it would be a good idea to set `swfAudioTrack = -1;` before each call to `updateAudioTrack` just so that in the assertions, we can know for sure the function worked the way we intended and not just because thats the track that was enabled previously.